RAG_OPTIMIZATION_GUIDE.md

```
@@ -0,0 +1,492 @@
+
+# If missing, run migration
+psql $DATABASE_URL -f migrations/add_hybrid_search_functions.sql
+```
+
+### Issue: "Reranking failed" warning
+
+**Causes**:
+1. `sentence-transformers` not installed
+2. Model not downloaded
+3. Out of memory (model requires ~1GB RAM)
+
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)